### PR TITLE
update to readme and structure so matches the manifesto

### PR DESCRIPTION
**Pull request by CarolineMorton**
update to readme to fit with manifesto document [here](https://docs.google.com/document/d/1LD5hVjFOWx1AptbXkdTS135ureLkxd8kCumgl8mxzaA/)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,11 +37,46 @@ folder by launching the notebook in the free online service,
...
+the Python is separated into a separate module, in `code/`, and
+imported from the notebook.
 
-XXX
+Data, including raw and processed data, should be stored within the `data/` folder. Paths can be created easily with 
+the `os` module. For more information, see [teaching resources](). 
 
-# Developing notebooks
+Tests live in `tests/` folder. Tests are run automatically with [pytest](https://docs.pytest.org/en/latest/). This library
+will find any python files called `test_*.py` and then find any functions called `test_*()`.
```
Comment (sebbacon) [inline README.md]: (and also verify notebooks are runnable with identical output, q.v)

**Review by sebbacon — COMMENTED** [review]
Great improvements!

I'm wondering if we should be pulling out some of this into a separate README for developers?

The thing that made me think this in particular was the "how to cite" thing which is currently written for developers and could be called "how to invite people to cite". Do you see what I mean?

**Comment by CarolineMorton** [comment]
Ok i have added those bits in. Are you happy to merge? @sebbacon

**Comment by sebbacon** [comment]
Yes! But I think separately we should think about splitting into `DEVELOPERS.md` (for developers) and `README.md` (for public).

**Comment by CarolineMorton** [comment]
Great. I think that is a good idea too. Sorry not to reply to that section specifically.

**Pull request merged**